### DISCUSSION TITLE
怎么添加ghcr.io的私有仓库，Gtihub只提供auth token，页面只能输入用户名密码？

### DISCUSSION THREAD

**Discussion by mxdlzg** [1Panel-dev/1Panel#4611] [Q&A]
只能手动在 config里配置吗？{
    "auths": {
        "ghcr.io": {
            "auth": "xxx"
        }
    }
}

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，已经添加为 github issue，我们会在后续版本考虑支持。
